scripts/enable.sh

```diff
@@ -77,8 +77,15 @@ else
     compose_up="false"
 fi
 
-curl -L https://github.com/docker/compose/releases/download/1.1.0-rc2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
-chmod +x /usr/local/bin/docker-compose
+if [ $distrib_id == "CoreOS" ]; then
```

Comment:
can we move this to `install.sh` ? this is more of installation than enabling. The `compose up -d` below is pretty much the enable step.